src/rhos/correlatorUtils.py

```diff
@@ -2,8 +2,8 @@
 import sys
 import math
 
-sys.path.append("../utils")
-from rhoUtils import *
+sys.path.append("..")
+from rhos.rhoUtils import *
```

Comment:
Use a relative import here. (I won't point out every instance of this, but watch out for it.)